src/cli/Arguments.zig

```diff
@@ -66,6 +66,7 @@ pub const transpiler_params_ = [_]ParamType{
     clap.parseParam("--tsconfig-override <STR>          Specify custom tsconfig.json. Default <d>$cwd<r>/tsconfig.json") catch unreachable,
     clap.parseParam("-d, --define <STR>...              Substitute K:V while parsing, e.g. --define process.env.NODE_ENV:\"development\". Values are parsed as JSON.") catch unreachable,
     clap.parseParam("--drop <STR>...                   Remove function calls, e.g. --drop=console removes all console.* calls.") catch unreachable,
+    clap.parseParam("--feature <STR>...               Enable a feature flag for dead-code elimination, e.g. --feature=SUPER_SECRET") catch unreachable,
```

Comment:
add `features` to JSBundler.zig Options read "features: string[]"  and wire it up to work in bundle_v2.zig so that this feature is available through Bun.build. Use expectBundled for those tests and don't forget to update itBundled.ts to propagate the features array properly. Don't forget to update tests using expectBundled to have both target: "cli" and target: "api" (probably a for loop over each) and don't forget to update bun.d.ts Bun.build types to highlight this feature